keep-only-warm-data/README.md

```diff
@@ -0,0 +1,125 @@
+# Keep only warm data into Kuzzle
+
+## Requirements
+
+Kuzzle : `>= 1.2.11`  
+Elasticsearch : `>= 5.4.1`  
+Redis : `>= 3.2`  
+
+## Introduction
+
+Kuzzle is capable of managing a very large number of documents and offering very good performances.  
+However in a Big Data context, it can be interesting to rationalize the cost of the infrastructure necessary to Kuzzle by keeping on Kuzzle only the relevant data.  
+Coupled with a secondary database, Kuzzle can be used in a Hot/Warm architecture where the most recent data is kept on Kuzzle and the oldest data in a secondary DBMS.  
```

Comment:
We do not keep "the oldest data in a secondary DBMS". Rather, we synchronize Kuzzle with another database (whatever this database's architecture might be)